cmd/api/src/api/v2/file_uploads.go

```diff
@@ -172,13 +193,20 @@ func (s Resources) ListAcceptedFileUploadTypes(response http.ResponseWriter, req
 	api.WriteBasicResponse(request.Context(), ingestModel.AllowedFileUploadTypes, http.StatusOK, response)
 }
 
-func IsValidContentTypeForUpload(header http.Header) bool {
+func ParseUploadContentType(header http.Header) (string, error) {
```

Comment:
This should probably be private, I'm not sure why the previous was public tbh, but we can fix that here.